### DISCUSSION TITLE
Add compatibility matrix in readme and/or in release notes

### DISCUSSION THREAD

**Discussion by jtournemine** [eviden-parallel/community#19] [Ideas]
It would be nice to explain :
- Which web browsers and their respective versions are supported
- What are the min and max screen size and/or resolution supported

**Comment by Th3S4mur41** [comment]
Hi @jtournemine 

# Browser support

Browser support can be easily extracted from the `.browserslist`

## Desktop
last 3 Chrome versions
last 3 Edge versions
last 3 Firefox versions
Firefox ESR
last 2 Opera versions
last 2 Safari major versions

## Mobile
last 2 ChromeAndroid versions
last 2 FirefoxAndroid versions
last 2 iOS versions

# Screen size

Screen size is a bit more difficult to answer.
All our components are responsive and should adapt to their environment.
So far we do design and test for mobile to desktop screens, so from this perspective we could say:

**Minimun screen size: [mobile interstitial (320×480)](https://viewportsizer.com/phone-screen-dimensions/)**

I'm not sure defining a maximum screen size makes as much sense though?! 

I mean the worst that could happen, is that a component might grow indefinitely to the screen size and not look nice, but it will not break the same way it would with a small screen...

Also, even if we were to say e.g. **4K**, what would that mean in term of screen format. Is that 3:2 or regular 16:9 or 16:10.
But then what about UltraWide screens?
And what about future **8K** screens?

@normanatos @fdroess any thoughts on this from a design perspective?

https://github.com/eviden-parallel/docs/issues/137 created to document this
